### PR TITLE
Relative folder path for related files info

### PR DESCRIPTION
**Pull request by jsonperl**

### CODE REVIEW

File: lib/bibliothecary/related_files_info.rb [hunk 1]
```diff
@@ -20,9 +21,12 @@ def self.create_from_file_infos(file_infos)
     def initialize(file_infos)
       package_manager = file_infos.first.package_manager
       @platform = package_manager.platform_name
-      @folder_path = File.join(file_infos.first.folder_path, File.dirname(file_infos.first.relative_path))
+      file_infos.first.tap do |initial|
+        @folder_path = Pathname.new(initial.folder_path).join(File.dirname(initial.relative_path)).cleanpath.to_path
+        @relative_folder_path = Pathname.new(File.dirname(initial.relative_path)).cleanpath.to_path
+      end
```
Comment (havocp) [inline lib/bibliothecary/related_files_info.rb]: something about the naming here is confusing - a comment with examples would be pretty helpful I think.

Comment (jsonperl) [reply lib/bibliothecary/related_files_info.rb]: I'm pretty confused myself (also about fileinfo itself). I'm wondering if we should ditch both and just add a "path". I think the problem is that it's basing it on the root path of the project it's run in, and that doesn't seem useful if you're scanning a subpath.

**Comment by jsonperl** [comment]
@mellisio / @havocp SImplified this to just #path. Watcha think?

File: lib/bibliothecary/related_files_info.rb [hunk 2]
```diff
@@ -1,7 +1,6 @@
 module Bibliothecary
   class RelatedFilesInfo
-    attr_reader :folder_path
-    attr_reader :relative_folder_path
+    attr_reader :path
```
Comment (havocp) [inline lib/bibliothecary/related_files_info.rb]: so which path is this? the relative subfolder path that the related files are inside?

Comment (havocp) [reply lib/bibliothecary/related_files_info.rb]: if so maybe leave it named folder_path but make it always relative (don't need the absolute one at all?)

Comment (jsonperl) [reply lib/bibliothecary/related_files_info.rb]: Yep. I figure you called the thing with the base path of what you're looking at, so you would expected everything to be relative to that. (I would expect that)

Comment (jsonperl) [reply lib/bibliothecary/related_files_info.rb]: I'd agree if we were looking at a *file* but we're basically looking at a folder, so it feels sort of redundant. But I'm not super committed.

Comment (jsonperl) [reply lib/bibliothecary/related_files_info.rb]: "The *path* of the files"

Comment (havocp) [reply lib/bibliothecary/related_files_info.rb]: wfm :-) let's not block on over-perfecting this

**Pull request merged**